wpilibc/src/main/native/include/frc/experimental/controller/PIDController.h

```diff
@@ -0,0 +1,328 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <functional>
+#include <limits>
+
+#include <wpi/mutex.h>
+
+#include "frc/experimental/controller/Controller.h"
+#include "frc/experimental/controller/MeasurementSource.h"
+#include "frc/smartdashboard/SendableBase.h"
+
+namespace frc {
+namespace experimental {
+
+// TODO: remove when this class is moved into frc namespace
+using frc::SendableBase;
+using frc::SendableBuilder;
+
+/**
+ * Class implements a PID Control Loop.
+ *
+ * Creates a separate thread which reads the given PIDSource and takes care of
+ * the integral calculations, as well as writing the given PIDOutput.
+ *
+ * This feedback controller runs in discrete time, so time deltas are not used
+ * in the integral and derivative calculations. Therefore, the sample rate
+ * affects the controller's behavior for a given set of PID constants.
```

Comment:
Staring at this a bit more, isn't this documentation incorrect for this rewrite?